starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,25 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  if (first > second) {
+    return first;
+  } else {
+    return second;
+  }
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
 function findLongestWord (words) {
-
+  var longest = 0;
+  for(var i = 0; i<words.length; i++) {
+    var wordLength = words[i].length;
```

Comment:
you don't have to declare var wordLength: 

"     if (longest < words[i].length) {
       longest = words[i].length;
    }"